homeassistant/components/onvif/camera.py

```diff
@@ -256,6 +256,7 @@ def __init__(self, hass, config):
 
             _LOGGER.debug("Retrieving stream uri")
 
+            media_service = self._camera.create_media_service()
```

Comment:
Please add a code comment before this line, explaining why it's needed. This line could easily be removed and lost in a refactor without this knowledge.